src/ontology/mondo.Makefile

```diff
@@ -289,6 +289,38 @@ sources/$(SOURCE_VERSION)/equivalencies.owl: | source_release_dir
 #sources/CTD_diseases.obo:
 #	curl -L -s http://ctdbase.org/reports/CTD_diseases.obo.gz  | gzip -dc | perl -npe 's@alt_id@xref@' > $@.tmp && mv $@.tmp $@
 
+# TODO: Gard mapped obsoletes
+# Working helper commands ------
+tmp/mondo-edit.owl:
+	robot convert -i mondo-edit.obo --format owl -o $@
+tmp/mondo-edit.db: tmp/mondo-edit.owl
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+tmp/obsoletes.tsv: tmp/mondo-edit.db
+	runoak -i sqlite:tmp/mondo-edit.db obsoletes > tmp/obsoletes.tsv
+
+ofn:
+	robot convert -i mondo-edit.obo --format ofn -o tmp/mondo-edit.ofn
+
+# Failed commands ------
+#assocs:
+#	runoak -i sqlite:tmp/mondo-ingest.db associations -p i DOID:2757
+#axioms:
+#	runoak -i tmp/mondo-edit-temp.ofn axioms –axiom-type MONDO:equivalentTo
+#	runoak -i tmp/mondo-edit-temp.ofn axioms –axiom-type SubClassOf
+
+# todo: eventually
+#tmp/obsoletes_gard.tsv: tmp/obsoletes.tsv
+##	runoak -i sqlite:tmp/mondo-ingest.db obsoletes > tmp/obsoletes.tsv
+##	runoak -i sqlite:tmp/mondo-ingest.db obsoletes | grep "GARD:" > tmp/obsolete_gard.tsv  # 0 results; but above line has many
+
+# todo: find something that helps
+relat:
+	runoak -i sqlite:tmp/mondo-edit.db relationships -p i MONDO:0000004 --predicates oboInOwl:hasDbXref skos:exactMatch skos:narrowMatch skos:broadMatch skos:relatedMatch
```

Comment:
### Failed attempt: `runoak relationships`
This relationship command does not show any results, even though the class has many such relationships:
```owl
    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0000004">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MONDO_0002816"/>
        <rdfs:subClassOf>
            <owl:Restriction>
                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0004024"/>
                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/GO_0034651"/>
            </owl:Restriction>
        </rdfs:subClassOf>
        <rdfs:subClassOf>
            <owl:Restriction>
                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0004026"/>
                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001235"/>
            </owl:Restriction>
        </rdfs:subClassOf>
        <obo:IAO_0000115>An endocrine or hormonal disorder that occurs when the adrenal cortex does not produce enough of the hormone cortisol and in some cases, the hormone aldosterone. It may be due to a disorder of the adrenal cortex (Addison&apos;s disease or primary adrenal insufficiency) or to inadequate secretion of ACTH by the pituitary gland (secondary adrenal insufficiency).</obo:IAO_0000115>
        <oboInOwl:hasDbXref>DOID:10493</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>GARD:0006722</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>ICD9:255.4</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>ICD9:255.41</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>MESH:D000309</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>NCIT:C26691</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SCTID:154707007</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SCTID:190527008</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SCTID:267398003</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SCTID:267483004</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SCTID:386584007</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SCTID:68588005</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>UMLS:C0001623</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>UMLS:C0405580</oboInOwl:hasDbXref>
        <oboInOwl:hasExactSynonym>adrenal cortical hypofunction</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>adrenal cortical insufficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>adrenal gland insufficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>adrenal insufficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>adrenocortical insufficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>corticoadrenal insufficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>hypocortisolemia</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>hypocortisolism</oboInOwl:hasExactSynonym>
        <oboInOwl:hasRelatedSynonym>hypoadrenalism</oboInOwl:hasRelatedSynonym>
        <oboInOwl:id>MONDO:0000004</oboInOwl:id>
        <rdfs:label>adrenocortical insufficiency</rdfs:label>
    </owl:Class>
```

I also tried without listing any `--predicates` explicitly, and it only showed `subClassOf`

Reply:
I created an issue in OAK: https://github.com/INCATools/ontology-access-kit/issues/613